src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -928,22 +928,44 @@ def download_fs(request):
         raise django.http.Http404
 
 
-def download_by_uuid(request, uuid):
+def preview_by_uuid(request, uuid):
+    """Simple wrapper to the download_by_uid function to create a route
+    through the code to enable the preview of a file in the browser, not
+    automatically download.
     """
-    Download a file from the Storage Service, given its UUID.
+    return download_by_uuid(request, uuid, True)
 
-    This view will stream the response directly from the storage service, so, unlike download_ss, this will work even if the Storage Service is not accessible to the requestor.
 
-    Returns 404 if a file with the requested UUID cannot be found, and 400 if the storage service fails to retrieve the record.
+def download_by_uuid(request, uuid, preview_file=False):
+    """Download a file from the Storage Service, given its UUID.
+
+    This view will stream the response directly from the storage service,
+    so, unlike download_ss, this will work even if the Storage Service is
+    not accessible to the requestor.
+
+    Returns 404 if a file with the requested UUID cannot be found, and
+    400 if the storage service fails to retrieve the record.
+
+    preview_file is an instruction to be applied to the response headers to
```

Comment:
I'm in the process myself of getting used to a pattern that Joel introduced: using reStructuredText in docstrings and things like rst inline literals, e.g.:

    ``preview_file`` is an introduction...

Reply:
I think this is a good idea. I'll try and remember it when adding snippets like this to comments. Q. Do our docstrings render anywhere? These can be easier to reinforce (like GoDocs) when there's something tangible to look at.